pitest-entry/src/main/java/org/pitest/process/Java9Process.java

```diff
@@ -108,8 +108,8 @@ private List<String> createLaunchArgs(JavaAgent agentJarLocator, List<String> ar
         List<String> cmd = new ArrayList<>();
 
         cmd.add("-classpath");
-        cmd.add(classPath);
-
+        cmd.add(classPath.replace(" ", "\" \""));
```

Comment:
I wonder, if it can affect the Gradle plugin, in the way how it already passes the classpath entries :thinking:

Reply:
It has just occurred to me that this change would cause a problem if gradle were already quoting the the spaces. Otherwise, this should have no impact.